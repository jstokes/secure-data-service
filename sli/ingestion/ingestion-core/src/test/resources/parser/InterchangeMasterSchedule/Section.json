{
  "a_id":"ID005",
  "UniqueSectionCode":
  {
    "_value":"UniqueSectionCode1"
  },
  "SequenceOfCourse":4,
  "EducationalEnvironment":
  {
    "_value":"Classroom"
  },
  "MediumOfInstruction":
  {
    "_value":"Televised"
  },
  "PopulationServed":
  {
    "_value":"Regular Students"
  },
  "AvailableCredit":
  {
    "a_CreditType":"Carnegie unit",
    "a_CreditConversion":0,
    "Credit":
    {
      "_value":50
    }
  },
  "CourseOfferingReference":
  {
    "CourseOfferingIdentity":
    {
      "LocalCourseCode":
      {
        "_value":"LocalCourseCode3"
      },
      "SessionReference":
      {
        "SessionIdentity":
        {
          "SessionName":
          {
            "_value":"SessionName4"
          }
        }
      },
      "EducationalOrgReference":
      {
        "EducationalOrgIdentity":
        {
          "StateOrganizationId":
          {
            "_value":"StateOrganizationId12"
          }
        }
      }
    }
  },
  "SchoolReference":
  {
    "EducationalOrgIdentity":
    {
      "StateOrganizationId":
      {
        "_value":"StateOrganizationId13"
      }
    }
  },
  "SessionReference":
  {
    "SessionIdentity":
    {
      "EducationalOrgReference":
      {
        "EducationalOrgIdentity":
        {
          "StateOrganizationId":
          {
            "_value":"StateOrganizationId14"
          }
        }
      },
      "SessionName":
      {
        "_value":"SessionName5"
      }
    }
  },
  "LocationReference":
  {
    "a_id":"ID006",
    "LocationIdentity":
    {
      "ClassroomIdentificationCode":
      {
        "_value":"ClassroomIdentificat"
      },
      "EducationOrgIdentificationCode":
      [
        {
          "a_IdentificationSystem":"School",
          "ID":
          {
            "_value":"ID2"
          }
        },
        {
          "a_IdentificationSystem":"School",
          "ID":
          {
            "_value":"ID3"
          }
        },
        {
          "a_IdentificationSystem":"School",
          "ID":
          {
            "_value":"ID4"
          }
        },
        {
          "a_IdentificationSystem":"School",
          "ID":
          {
            "_value":"ID5"
          }
        }
      ]
    }
  },
  "ClassPeriodReference":
  {
    "a_id":"ID007",
    "ClassPeriodIdentity":
    {
      "ClassPeriodName":
      {
        "_value":"ClassPeriodName1"
      },
      "StateOrganizationId":
      [
      {
        "_value":"StateOrganizationId17"
      },
      {
        "_value":"StateOrganizationId18"
      }
      ]
    }
  },
  "ProgramReference":
  [
    {
      "ProgramIdentity":
      {
        "ProgramId":
        {
          "_value":"ProgramId2"
        }
      }
    },
    {
      "ProgramIdentity":
      {
        "ProgramId":
        {
          "_value":"ProgramId3"
        }
      }
    }
  ]
}