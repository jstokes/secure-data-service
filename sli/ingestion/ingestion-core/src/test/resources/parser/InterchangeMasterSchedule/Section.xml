<?xml version="1.0" encoding="UTF-8"?>
<InterchangeMasterSchedule xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/beta/sli/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-MasterSchedule.xsd">
    <Section id="ID001">
        <UniqueSectionCode>glt71G8aD34w16HEE9tfsV5ABEnnI2</UniqueSectionCode>
        <SequenceOfCourse>7</SequenceOfCourse>
        <EducationalEnvironment>Mainstream (Special Education)</EducationalEnvironment>
        <MediumOfInstruction>Other technology-based instruction</MediumOfInstruction>
        <PopulationServed>ESL Students</PopulationServed>
        <AvailableCredit CreditType="Semester hour credit" CreditConversion="918241612.14">
            <Credit>266982401.27</Credit>
        </AvailableCredit>
        <CourseOfferingReference>
            <CourseOfferingIdentity>
                <LocalCourseCode>WCv8XCMGy-rPgL</LocalCourseCode>
                <SessionReference>
                    <SessionIdentity>
                        <SessionName>dTiw5pEbayOq3bgDzfiqtNNrU_NR</SessionName>
                    </SessionIdentity>
                </SessionReference>
                <EducationalOrgReference>
                    <EducationalOrgIdentity>
                        <StateOrganizationId>yB2T2K8rw5s1x8</StateOrganizationId>
                    </EducationalOrgIdentity>
                </EducationalOrgReference>
            </CourseOfferingIdentity>
        </CourseOfferingReference>
        <SchoolReference>
            <EducationalOrgIdentity>
                <StateOrganizationId>oBGb</StateOrganizationId>
            </EducationalOrgIdentity>
        </SchoolReference>
        <SessionReference>
            <SessionIdentity>
                <EducationalOrgReference>
                    <EducationalOrgIdentity>
                        <StateOrganizationId>fuHocabJk2b_lZouywtP9QQ</StateOrganizationId>
                    </EducationalOrgIdentity>
                </EducationalOrgReference>
                <SessionName>kKetPH376YbHpXKuq9jUQX22qxtnEfek8Z</SessionName>
            </SessionIdentity>
        </SessionReference>
        <LocationReference id="ID003" ref="ID002">
            <LocationIdentity>
                <ClassroomIdentificationCode>ybmi89O2</ClassroomIdentificationCode>
                <StateOrganizationId>Eo6RxoIJtbMyG0Agz6Y03mWOeXfxLdCk03F4-_</StateOrganizationId>
            </LocationIdentity>
        </LocationReference>
        <ClassPeriodReference id="ID005" ref="ID005">
            <ClassPeriodIdentity>
                <ClassPeriodName>yfcdo</ClassPeriodName>
                <EducationOrgIdentificationCode IdentificationSystem="DUNS">
                    <ID>dGkcZpLxJ</ID>
                </EducationOrgIdentificationCode>
            </ClassPeriodIdentity>
        </ClassPeriodReference>
        <ProgramReference>
            <ProgramIdentity>
                <ProgramId>IJeOzO5HTVhF</ProgramId>
            </ProgramIdentity>
        </ProgramReference>
    </Section>
</InterchangeMasterSchedule>
