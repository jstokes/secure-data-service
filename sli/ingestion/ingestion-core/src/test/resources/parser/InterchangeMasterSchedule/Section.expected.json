{
  "@id":"ID001",
  "UniqueSectionCode":{"_value":"glt71G8aD34w16HEE9tfsV5ABEnnI2"},
  "SequenceOfCourse":7,
  "EducationalEnvironment":{"_value":"Mainstream (Special Education)"},
  "MediumOfInstruction":{"_value":"Other technology-based instruction"},
  "PopulationServed":{"_value":"ESL Students"},
  "AvailableCredit":{
    "@CreditType":"Semester hour credit",
    "@CreditConversion":9.1824161214E8,
    "Credit":2.6698240127E8
  },
  "CourseOfferingReference":{
    "CourseOfferingIdentity":{
      "LocalCourseCode":{"_value":"WCv8XCMGy-rPgL"},
      "SessionReference":{
        "SessionIdentity":{
          "SessionName":{"_value":"dTiw5pEbayOq3bgDzfiqtNNrU_NR"}
        }
      },
      "EducationalOrgReference":{
        "EducationalOrgIdentity":{
          "StateOrganizationId":{"_value":"yB2T2K8rw5s1x8"}
        }
      }
    }
  },
  "SchoolReference":{
    "EducationalOrgIdentity":{
      "StateOrganizationId":{"_value":"oBGb"}
    }
  },
  "SessionReference":{
    "SessionIdentity":{
      "EducationalOrgReference":{
        "EducationalOrgIdentity":{
          "StateOrganizationId":{"_value":"fuHocabJk2b_lZouywtP9QQ"}
        }
      },
      "SessionName":{"_value":"kKetPH376YbHpXKuq9jUQX22qxtnEfek8Z"}
    }
  },
  "LocationReference":{
    "@id":"ID003",
    "@ref":"ID002",
    "LocationIdentity":{
      "ClassroomIdentificationCode":{"_value":"ybmi89O2"},
      "StateOrganizationId":[{"_value":"Eo6RxoIJtbMyG0Agz6Y03mWOeXfxLdCk03F4-_"}]
    }
  },
  "ClassPeriodReference":{
    "@id":"ID005",
    "@ref":"ID005",
    "ClassPeriodIdentity":{
      "ClassPeriodName":{"_value":"yfcdo"},
      "EducationOrgIdentificationCode":[{
        "@IdentificationSystem":"DUNS",
        "ID":{"_value":"dGkcZpLxJ"}
      }]
    }
  },
  "ProgramReference":[{
    "ProgramIdentity":{
      "ProgramId":{"_value":"IJeOzO5HTVhF"}
    }
  }]
}